<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fsDialog</class>
 <widget class="QDialog" name="fsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Быстрый поиск пациента</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblQuery">
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;
&lt;head/&gt;
&lt;body style='font-family:Times; font-size:10pt; font-style:normal'&gt;
Шаблон быстрого поиска
&lt;/body&gt;
&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>edtQuery</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edtQuery"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnQuery">
        <property name="text">
         <string>Искать</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="lblShabl">
        <property name="text">
         <string>&lt;html&gt;
&lt;head/&gt;
&lt;body style='font-family:Times; font-size:11pt; font-style:normal'&gt;
Допустимые шаблоны поиска:&lt;br/&gt;
ФИОДДММГГГГ (РЕЕ09021995)&lt;br/&gt;
ФИОДДММГГ (РЕЕ090295)&lt;br/&gt;
ФИДДММГГГГ (РЕ09021995)&lt;br/&gt;
ФИДДММГГ (РЕ090295)&lt;br/&gt;
Ф* И* (Руд Его)&lt;br/&gt;
Ф* И* О* (Руден Ег Евген)&lt;br/&gt;
Ф* И* О* ДДММГГГГ (Руденк Егор Евг 09021995)&lt;br/&gt;
Ф* И* О* ДДММГГ (Руд Ег Евг 090295)&lt;br/&gt;
Ф* И* ДДММГГГГ (Руд Егор 09021995)&lt;br/&gt;
Ф* И* ДДММГГ (Руд Его 090295)&lt;br/&gt;
Ф* ДДММГГГГ (Руден 09021995)&lt;br/&gt;
Ф* ДДММГГ (Руден 090295)&lt;br/&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkWithMask">
        <property name="font">
         <font>
          <family>Times</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>По маске</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtQuery</tabstop>
  <tabstop>btnQuery</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
